app/assets/stylesheets/application.scss

```diff
@@ -15,28 +15,74 @@
 *= require foundation_and_overrides
 *= require_self
  */
+@import url(https://fonts.googleapis.com/css?family=Montaga);
+$lightgray: #ede0d5;
+$blue: #354a57;
 $borderbrown: #a52a2a;
 $backgroundorange: #ffffcc;
 $fontgreen: #008000;
+$fontblue: #94c9ff;
 
-.side-nav {
-  height: 100vh;
+
+.content {
+  padding-left: 20px;
+}
+.side-nav  {
+  background-color: #b94947;
```

Comment:
Color literals like `#b94947` should only be used in variable declarations; they should be referred to via variable everywhere else.